src/Aptoma/Twig/Extension/MarkdownEngine/ParsedownEngine.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Aptoma\Twig\Extension\MarkdownEngine;
+
+use Aptoma\Twig\Extension\MarkdownEngineInterface;
+use Parsedown;
+
+/**
+ * ParsedownEngine.php
+ *
+ * Maps erusev/parsedown to Aptoma\Twig Markdown Extension
+ *
+ * @author Sébastien Lourseau <https://github.com/SebLours>
+ */
+class ParsedownEngine implements MarkdownEngineInterface
+{
+    /**
+     * Parsedown instance name.
+     */
+    const INSTANCE_NAME = 'atoma';
+
+    /**
+     * @var Parsedown
+     */
+    protected $engine;
+
+    /**
+     * Constructor.
+     */
+    public function __construct()
+    {
+        $this->engine = Parsedown::instance(self::INSTANCE_NAME);
```

Comment:
Wouldn't it be better to take the instance name as a constructor parameter, and just use the default if it's not provided?